src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -5208,8 +5208,20 @@ def _build_arguments_schema(cls, *args, **kwargs):
             template="/subscriptions/{subscription}/resourceGroups/{resource_group}/providers/Microsoft.Network"
                      "/ddosProtectionPlans/{}",
         )
+        args_schema.ip_tags = AAZDictArg(
+            options=["--ip-tags"],
+            help="Space-separated list of IP tags in `TYPE=VAL` format.",
+        )
```

Comment:
maybe we can support `null` value for ip_tags so that customers can clean up this property.